src/app/app.component.html

```diff
@@ -38,31 +38,45 @@ <h1 class="color2 header2">Let’s find some clinical trials</h1>
         <h1 class="color1 subText"> We found your <span class="color12" (click)="showRecord();">record</span>. Just fill
           in some preferences and
           we’ll do the rest</h1>
+        <div class="containingDIV col100" style="padding-right: 10%;">
+          <div style="width: 23.5%;">
+            <h1 class="color1">Zip Code</h1>
+          </div>
+          <div style="width: 23.5%;">
+            <h1 class="color1">Travel Distance</h1>
+          </div>
+          <div style="width: 23.5%;">
+            <h1 class="color1">Trial Phase</h1>
+          </div>
+          <div style="width: 23.5%;">
+            <h1 class="color1">Recruitment Status</h1>
+          </div>
+        </div>
         <div class="containingDIV col100" style="padding-right: 10%;">
           <div class="block">
             <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24.5 35">
               <path id="ic_place_24px"
                 d="M17.25,2A12.241,12.241,0,0,0,5,14.25C5,23.437,17.25,37,17.25,37S29.5,23.437,29.5,14.25A12.241,12.241,0,0,0,17.25,2Zm0,16.625a4.375,4.375,0,1,1,4.375-4.375A4.377,4.377,0,0,1,17.25,18.625Z"
                 transform="translate(-5 -2)" fill="#525151" />
             </svg>
-            <input class="input" type="text" placeholder="zip code or city" [(ngModel)]="searchReqObject.zipCode">
+            <input class="input" type="text" placeholder="zip code or city" [(ngModel)]="searchReqObject.zipCode" (keyup.enter)="searchClinicalTrials(null);">
```

Comment:
My one issue with doing it this way instead of wrapping in a form and using the `submit` is that I suspect this won't work as expected if used via a mobile browser. Then again, I'm not sure we support that *at all* right now.